Checking for 'const overload' fails with 'unused variable i',
resulting in build errors (suggesting -fpermissive) with gcc-4.6:
http://bugs.gentoo.org/show_bug.cgi?id=365117

http://sourceforge.net/mailarchive/forum.php?thread_name=4DDFA16F.6040109%40gentoo.org&forum_name=mico-devel
--- mico/aclocal.m4
+++ mico/aclocal.m4
@@ -353,11 +353,12 @@
 
   void bar (const int &, int &, const int *, int *, int *&);
 
-  void foo (S &s, const S &cs)
+  int foo (S &s, const S &cs)
   {
-     int i = s; i = cs; i = *s; i = *cs;
+     int i = s; int j = cs; int k = *s; int l = *cs;
      bar (s, s, s, s, s);
      bar (cs, s, cs, s, s);
+     return (i + j + k + l);
   }
 ],
 eval "ac_cv_have_const_overload=yes",
